chart/openfaas/README.md

```diff
@@ -281,19 +281,6 @@ By default services will be exposed with following hostnames (can be changed, se
 
 * `gateway.openfaas.local`
 
-### Endpoint load-balancing
```

Comment:
Not sure why this would be deleted?

Reply:
The purpose of disabling direct functions is so that load balancing behaves as expected, it is even one of the options of the section. I wasn't sure how to rewrite this section because I wasn't sure what the goal/problem it would be trying to solve.

Reply:
Linkerd needs to be mentioned and Istio and how they need this setting disabled.

Reply:
Just added a new section